database scripts - mysql/Itembank/Stored Procedures/updatetdsconfigs.sql

```diff
@@ -60,7 +60,13 @@ begin
 		select tpp.propvalue 
 		from loader_testpackageproperties tpp 
 		where tpp.propname = 'msb'), '0') = '1' then   
-			update configs.client_testproperties ctp set msb = 1 where ctp.testid = v_testname;
+			update configs.client_testproperties ctp set msb = 1 
```

Comment:
Are we ok with making the first update more restrictive than it was prior to this change?  Specifically adding the `ct.clientname`?